topic04-git-introduction/book-git-tutorial/01.Basics.md

```diff
@@ -0,0 +1,30 @@
+# **_Git guide to commands_**
+
+This manual contains necessary commands for git when working on your own, contributing to other projects and working as a team.
+
+###### What is git about?
+
+Git is a network that allows you to manage your code easily while working as a team or contributing to other repos, It allows you to merge code without having to copy and paste it from another file or files. Git also is able to host your website but it does not give you a proper full domain. 
+
+You are also able to use it to fork a repo which creates a copy of a repo in your github account and allows you to do pull requests which allow you to basically create requests that allow you to suggest changes to the original owner of the repo which they can review, accept or decline and if they accept they can merge it into their original repo.
+
+#### **_The basics:_**
+`Git push`
+`Git pull`
+`Git add (filename)`
+`Git status`
+`Git commit -m “first commit”` 
+
+- These commands are used to add files to your repository through a terminal or git bash.
+- `git pull`, fetches changes from the remote master and merges changes, it can also be used like this `git pull (remote) (branchname)`.

```

Comment:
would add an example here too of an actual git pull (remote) (branchname) command